Popular_Models_Guide/Llama2/trtllm_guide.md

```diff
@@ -36,16 +36,20 @@ You will need to get permissions for the Llama2 repository as well as get access
 
 1. The installation starts with cloning the TensorRT-LLM Backend and update the TensorRT-LLM submodule:
 ```bash
-git clone https://github.com/triton-inference-server/tensorrtllm_backend.git
+git clone https://github.com/triton-inference-server/tensorrtllm_backend.git  --branch <release branch>
 # Update the submodules
 cd tensorrtllm_backend
+# Change submodule remote url to use https if necessary
+git submodule set-url -- tensorrt_llm https://github.com/NVIDIA/TensorRT-LLM.git
 git submodule update --init --recursive
 git lfs install
 git lfs pull
```

Comment:
They change the submodule url lately, so we won't need to set the remote url anymore. Please see below for the updated commands for updating submodule.
```suggestion
# Install git-lfs if needed
apt-get update && apt-get install git-lfs -y --no-install-recommends
git lfs install
git submodule update --init --recursive
```